.travis.yml

```diff
@@ -2,23 +2,37 @@ sudo: false
 
 language: php
 
-php:
-    - 7.2
-
-branches:
-    only:
-        - master
-
-cache:
-    directories:
-        - $HOME/.composer/cache/files
+php: 7.2
 
 before_install:
-    - phpenv config-rm xdebug.ini
+    - phpenv config-rm xdebug.ini || true
 
 install:
     - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
 
-script:
-    - vendor/bin/phpunit
-    - vendor/bin/phpcs
+jobs:
+    include:
+
+        - stage: Test
+          script:
+          - vendor/bin/phpunit
```

Comment:
is the different indentation of `script:` contents in the two stages intended? I find the Travis definitions of stages asymmetric at best